portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,20 +26,29 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> comments = new ArrayList<String>();
```

Comment:
Would be good to declare the member variable as a more generic type:

List<String> comments = new ArrayList<String>();

Reply:
Will do!